main.js

```diff
@@ -0,0 +1,91 @@
+// after document is ready
+$(document).ready(function () {
+  // add post button
+  $('.add-post').on('click', function (e) {
```

Comment:
This is a lot of code to have inside of one click handler - it'd be nice to have multiple, small functions that are each in charge of something specific. That would make this code easier to read.